docs/_static/vdr/v1.yaml

```diff
@@ -362,4 +378,44 @@ components:
         type:
           description: The type of the key.
           example: "JsonWebKey2020"
-          type: string
\ No newline at end of file
+          type: string
+    DIDCreateRequest:
+      properties:
+        controllers:
+          description: |
+            List of DIDs that can control the new DID Document. If selfControl = true and controllers is not empty.
```

Comment:
second sentence ends with a dot but should probably end with a comma?

Reply:
indeed